### DISCUSSION TITLE
can we create any conceptual database (like some projectname) to segregate collections in different database.

### DISCUSSION THREAD

**Discussion by ChaturyaLomada** [milvus-io/milvus#25311] [Q&A and General discussion]
I have installed milvus along with attu using helm.
Now in my backend created one database other than default and create few collections in that newly created database.
In my milvus UI unable to see that newly created database and the collections which I have created in that db.
Only able to see the collections created in default db in my Milvus UI.

**Comment by yhmo** [comment]
Currently, the Attu has no ability to show a non-default database. We will make this enhancement later.
